cmd/template/cluster/provider/capa.go

```diff
@@ -109,7 +109,12 @@ func templateClusterAWS(ctx context.Context, k8sClient k8sclient.Interface, outp
 
 			for i := 0; i < subnetCount; i++ {
 				flagValues.Network.Subnets = append(flagValues.Network.Subnets, capa.Subnet{
-					CidrBlock: subnets[i].CIDR().String(),
+					CidrBlocks: []capa.CIDRBlock{
+						{
+							CIDR:             subnets[i].CIDR().String(),
+							AvailabilityZone: fmt.Sprintf(string('a' - 1 + (1 + i))), // generate `a`, `b`, etc. based on which index we're at
```

Comment:
black magic

Reply:
what is teh point of removing 1 and then adding it? :p

Reply:
`mt.Sprintf(string('a'  + i))` is that enought?

Reply:
Yeah, you're right. I copy&paste from internet before going to get lunch :p 

Just updated it now and hopefully fixed lint complaints too